### DISCUSSION TITLE
undefined symbol: __asan_option_detect_stack_use_after_retur

### DISCUSSION THREAD

**Discussion by manjunatha-shivaswamy** [tokio-rs/tokio#5467] [Q&A]
Hi, 
With tokio-1.25.0, the following undefined symbol error seen: 
rror: /src/fuzz/target/debug/deps/libtokio_macros-062602394c58529f.so: undefined symbol: __asan_option_detect_stack_use_after_return
   --> /opt/rust/registry/src/crates.microsoft.com-529af63c8b22dae2/tokio-1.25.0/src/lib.rs:589:1

589 |     pub use tokio_macros::select_priv_declare_output_enum;
    |             ^^^^^^^^^^^^

If any workaround, please help. 

Thanks,

**Comment by Darksonn** [comment]
This sounds like a problem with your machine rather than a problem with Tokio. I don't know what you can do about it.
